src/goto/storage.py

```diff
@@ -207,7 +213,7 @@ def remove_teleport(name):
     '''Removes a teleport from the currently active profile.'''
     data = get_active_profile()
     if not data.get(name):
-        msg = '{} is not a location you can teleport to'.format(name)
+        msg = 'Error: {} is not a location you can teleport to'.format(name)
```

Comment:
I don't think it should say "Error:" In all error messages. That was probably unclear from my side in issue #26... Better to keep them like they are without "Error:" at the start.